algebra/group.lean

```diff
@@ -479,22 +479,31 @@ section add_comm_group
 
 end add_comm_group
 
+variables  [group α] {β : Type*} [group β]
```

Comment:
Is there a reason why you want to have it in this order?
Usually the order is: types, type classes instances, values, proofs. This does not always work due to dependencies. But in this case it should be no difference.